src/backend/base/langflow/components/vectorstores/astradb.py

```diff
@@ -805,6 +857,7 @@ async def _create_new_database(self, build_config: dict, field_value: dict) -> N
                 "status": "PENDING",
```

Comment:
you could store the result of the `await self.create_database_api(...)` which is an `AsyncDatabase`, and here read its values for real with its `.info()` method and the various attributes of the return value. But again, not very relevant.

Reply:
I'm going to also open a JIRA on this because that's a fantastic idea but i think i'll defer it to a follow up PR after hybrid search just to avoid adding too much in one here... thanks @hemidactylus !